configs/offsets/std_inspect.go

```diff
@@ -69,7 +70,23 @@ func regularGetRequest(ctx context.Context, url string) error {
 	return nil
 }
 
+func useDatabaseSQL() error {
```

Comment:
What about progams in config/offsets for the mysql and postgres drivers? I think we need those so that we can generate the offsets in tracker_input